blockreassurance.php

```diff
@@ -100,7 +107,7 @@ public function __construct()
             $this->context->link = new Link($protocolPrefix, $protocolPrefix);
         }
 
-        $this->displayName = $this->trans('blockreassurance', array(), 'Modules.Blockreassurance.Admin');
+        $this->displayName = $this->trans('Customer Reassurance', array(), 'Modules.Blockreassurance.Admin');
```

Comment:
Then you're gonna have to update the translation in the module right? Or is it a native module?

Reply:
Right now, it is a native module. But it might be turned into an API pushed module for a V2, we haven't made up our mind yet with @colinegin, we need to check a few elements before.

Reply:
So what do I do for this point ?

Reply:
Consider it as a native module. :-)